clients/fides-js/src/components/Overlay.tsx

```diff
@@ -110,19 +121,20 @@ const Overlay: FunctionComponent<Props> = ({
           options.debug,
           "Modal link element found, updating it to show and trigger modal on click."
         );
+        setIsModalLinkFound(true);
         // Update modal link to trigger modal on click
         const modalLink = modalLinkEl;
-        modalLink.onclick = () => {
-          setBannerIsOpen(false);
-          handleOpenModal();
-        };
+        modalLink.onclick = window.Fides.showModal;
```

Comment:
👍 

This is pretty much what I was imagining, yup!

As a slight bonus to this... I think we should consider switching this to use `addEventListener("click", (evt) => { ... })` instead of `onclick = () => { ... }`. This isn't the core ask of the ticket, but looking at this code again reminds me that, in general `onclick` feels fragile.